<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>94</width>
    <height>16</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign Up</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #27374D
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Alegreya Sans SC</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: #DDE6ED;
}</string>
    </property>
    <property name="text">
     <string>SIGN UP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="newUsername">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-radius: 12px;
	border: 1px solid #DDE6ED;
	background-color: #9DB2BF;
	padding-left: 10px;
	padding-right: 10px;
}

QLineEdit::focus {
	border: 1px solid #CD1818;
}
</string>
    </property>
    <property name="placeholderText">
     <string>Enter New Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPassword">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-radius: 12px;
	border: 1px solid #DDE6ED;
	background-color: #9DB2BF;
	padding-left: 10px;
	padding-right: 10px;
}

QLineEdit::focus {
	border: 1px solid #CD1818;
}
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Enter New Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPasswordConfirm">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Historic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border-radius: 12px;
	border: 1px solid #DDE6ED;
	background-color: #9DB2BF;
	padding-left: 10px;
	padding-right: 10px;
}

QLineEdit::focus {
	border: 1px solid #CD1818;
}
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm New Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterButton_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-radius: 9px;
	background-color: #CD1818;
	color: #DDE6ED;
}

QPushButton::hover {
	background-color: #27374D;
	border: 2px solid #DDE6ED;
}</string>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
